recipes/binny/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "binny" %}
+{% set version = "2.2.18" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ohickl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: bfbf93c70f71236d2aa3d7289accea3306651359c0e58fbe9902441d2b6f439f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: False
+  entry_points:
+    - binny = binny.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.11
+  run:
+    - python >=3.6,<3.11
+    - networkx =3.1.0
+    - snakemake =7.19.0
+    - pyarrow =12.0.1
+    - hdbscan =0.8.33
+    - scikit-bio =0.5.9
+    - opentsne =1.0.0
+    - seqkit =2.5.1
+    - mantis_pfa =1.5.5
+    - bedtools =2.31.0
+    - gcc_linux-64
+    - singularity =3.8.6
+    - git
+
+test:
+  commands:
+    - binny --help
+
+about:
+  home: https://github.com/a-h-b/binny
+  license: GPL-3
+  license_family: GPL
+  summary: '!!! TEST PACKAGE !!! An automated binning algorithm to recover high-quality genomes from complex metagenomic datasets. !!! TEST PACKAGE !!!'
```

Comment:
This doesn't look good ☺️

Can you please also use spdx identifiers for your license? Thanks!

Reply:
> This doesn't look good ☺️

Could you specify?

> Can you please also use spdx identifiers for your license? Thanks!

Like this:

`license:  GPL-3.0-or-later` ?

Reply:
Yes